package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jupiterone/typescript-tools",
-  "version": "14.0.2",
+  "version": "15.0.0",
```

Comment:
This was a major version change but the https://github.com/JupiterOne/typescript-tools/blob/main/CHANGELOG.md was not updated. Can we just upgrade without anything breaking/needing to change?